package.json

```diff
@@ -78,6 +78,7 @@
     "sinon": "^1.15.4",
     "sinon-chai": "^2.8.0",
     "transfer-webpack-plugin": "^0.1.4",
+    "warning": "^2.1.0",
```

Comment:
We have two dependencies right now https://github.com/callemall/material-ui/blob/master/package.json#L32